### ISSUE TITLE
Purchase confirmation email not sent for successful purchase

### ISSUE THREAD

**Issue by cbwinchild** [Tizra/Tizra-Customer-Tracker#365]
on cbw.tizrapublisher.com i made a 10 cent purchase using tizra fulfiller and did not receive purchase confirmation email. LIcense to the content was added successfully. I've copied the mail.addlicense and mail.packagepurchase templates being used on the site:

mail.addlicense:
[#ftl /]
To: ${emailAddress}
From: ${catherine.winchild@tizra.com}
Subject: New subscription for user ${emailAddress} at ${site.hostname}
Html: true
<html>

<p>Your account at ${site.hostname} has a new subscription to
"${product}". You can access it at <a href="${site.homepage}/${productUrl}">${site.homepage}/${productUrl}</a>.
If you want to see the details of your subscription, go to ${site.homepage}/~userInfo and log into
your account.
</p>

[#ftl /]
[#-- message parameters:
            ${passwordHelpUrl}
            ${accountInfoUrl}
            ${siteName}
            ${emailAddress} the user's email address
            ${totalAmount}
            ${totalTaxAmount}
            ${cardDetails}
# list offerInfoList as item:

```
        ${item.name}
        ${item.productName}
        ${item.productUrl}
        ${item.productPrice}
        ${item.expirationTerms}
```

  add a line like this to the header section of the mail, if you wish site admins to get copies:
Cc: email.admins
      --]

[!-- The following 3 lines are needed for the system to support some old sites, but can be removed from
new custom templates --]
[#if extraAddress??]
Cc: ${extraAddress}
[/#if]

mail.packagepurchase:
From: ${address.supportAddress}
To: ${emailAddress}
Bcc: ${address.supportAddress}
Subject: Your ${site.name} purchase
Html: true
<html><body><p>Thank you for purchasing online publications from ${siteName}. Details are below.</p>

<ul>
[#list offerInfoList as item]
    <li>Purchase of ${item.productName} for ${item.productPrice}<br />
    Access Term: ${item.accessTerms} <br />
    View purchased content: <a href="${item.productUrl}">${item.productUrl}</a>
    </li>
[/#list]
</ul>

[#if cardDetails??]

<p>Card transaction details: ${cardDetails}.</p>

[/#if]

<p>A username and password are required for access to the site. Your username is ${userName}. If you forget your
password, you can reset it at:</p>

<p><a href="${passwordHelpUrl}">${passwordHelpUrl}</a></p>

<p>To see a complete list of your online purchases, and manage your account information, go to...</p>

<p><a href="${accountInfoUrl}">${accountInfoUrl}</a></p>

<p>Please bookmark these pages and save this email so you can easily find your online purchases in the future.</p>

<p>If you have any questions or encounter problems, please do not hesitate to contact us at <a href="mailto:${address.supportAddress}">${address.supportAddress}</a></p>

<p style="font-size: .5em">Fine Print transaction details. Include in case of problems. ${transactionInformation}</p>

</body><html>

**Comment by daviddurand** [comment]
The email template you included has an error in it
`${catherine.winchild@tizra.com}` should just be `catherine.winchild@tizra.com` as the email address is not a variable.

The _big_ issue is that subsequent updates to the template are not taking effect until a system restart. So fixing the problem did not have any immediate effect on the email behavior. We will fix this caching problem, but in the meantime, we can schedule a restart if a template update needs to be made.

**Comment by cbwinchild** [comment]
Great. Thaks. Follow up question: if Dibble wants to make changes to their mail.purchasepackage email template, can I tell them to go ahead and make those changes and we'll schedule a restart?

**Comment by daviddurand** [comment]
This was fixed by Tizra/cubchicken@14d6be73d9c7473de74c954aed2bf2e1f9f8b167
